cmd/clusterawsadm/cmd/bootstrap/iam/cloudformation.go

```diff
@@ -156,7 +160,7 @@ func deleteCloudFormationStackCmd() *cobra.Command {
 				return err
 			}
 
-			return cfnSvc.ShowStackResources(t.Spec.StackName)
+			return nil
```

Comment:
We can get a signal from using `WaitUntilStackDeleteComplete ` here, similar to this:
```
klog.V(2).Infof("waiting for stack %q to delete", stackName)
	if err := s.CFN.WaitUntilStackDeleteComplete(&cfn.DescribeStacksInput{StackName: aws.String(stackName)}); err != nil {
		return errors.Wrap(err, "failed to delete AWS CloudFormation stack")
	}
```

Reply:
This is already called as part of `DeleteStack` here https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/cmd/clusterawsadm/cmd/bootstrap/iam/cloudformation.go#L153 https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/5a1266bcf509a4827c1d6d452cf10936bedc44be/cmd/clusterawsadm/cloudformation/service/service.go#L128 so it seems kind of redundant but I can add it if you feel strong about it.

Reply:
You are right, it is already called.